app/views/layouts/application.html.haml

```diff
@@ -8,7 +8,6 @@
     = javascript_include_tag 'application', 'data-turbolinks-track': 'reload'
   %body
     = render 'layouts/header'
-    %p.notice= notice
-    %p.alert= alert
+    - message(flash)
```

Comment:
does displaying this message work fine? :) I think we should use `=` instead of `-` to display executed output

Reply:
Works fine with - :D If I use = there is only {} instead content of message :o